packages/syft/src/syft/lib/sympc/session_util.py

```diff
@@ -24,19 +24,20 @@ def protobuf_session_serializer(session: Session) -> MPCSession_PB:
     length_rs = session.ring_size.bit_length()
     rs_bytes = session.ring_size.to_bytes((length_rs + 7) // 8, byteorder="big")
 
-    length_nr_parties = session.nr_parties.bit_length()
-    nr_parties_bytes = session.nr_parties.to_bytes(
-        (length_nr_parties + 7) // 8, byteorder="big"
-    )
+    # length_nr_parties = session.nr_parties.bit_length()
+    # nr_parties_bytes = session.nr_parties.to_bytes(
+    #     (length_nr_parties + 7) // 8, byteorder="big"
```

Comment:
MAYBE : We could remove the comments here.

Reply:
Done